### ISSUE TITLE
Failed to create new environment

### ISSUE THREAD

**Issue by nabilzhafri** [aws/copilot-cli#2863]
I was trying to create new environment to an existing app (which has been running great for the past few months), however I encountered this error when trying to create new environment today:

```
✘ Failed to link account ACCOUNT_ID and region eu-west-2 to application MY_APP.

✘ deploy env test to application MY_APP: adding test environment resources to application: operation 21 for stack set MY_APP-infrastructure failed
```

I ran this command:
```
cop env init --name test --profile MY_PROFILE --app MY_APP
```

I checked cloudformation, it mentioned this error:
`Unable to retrieve Arn attribute for AWS::ECR::Repository, with error message null`

But I'm not sure how to fix this error. I don't remember doing any customization, so I expect the command should've worked.

I've tried multiple times, but the issue seems persist. Any idea why?

**Comment by efekarakus** [comment]
Hi @nabilzhafri ! I'm so sorry for the inconvenience  🙇 , the error message seems so strange!

Since I see the following:
> operation 21 for stack set MY_APP-infrastructure failed

I wonder if the stackset operation could give us some additional troubleshooting information.

1. Would you mind navigating to your application stack set in the AWS CloudFormation console:

![stacksets](https://user-images.githubusercontent.com/879348/134381434-1f03e44a-0c0f-4388-ab72-036a9948bb96.png)

2. Clicking under the Operations tab, is it possible to check the Status of the operation 21 to see what was the reason?

![operations](https://user-images.githubusercontent.com/879348/134381883-f25b606a-634e-4322-9bc6-7a802a184211.png)

Thanks!

**Comment by nabilzhafri** [comment]
@efekarakus hi thanks for your reply.

I think I've finally figured it out. Apparently I have a service (that we don't use) but there's no ecr repo for that particular service (can't remember how did this happen - either the ecr repo was manually deleted, or the service was partially created)

Either way, I deleted that service, and now it works again.

So I guess what happened was, it couldn't find ecr report for that service, which is what the error was about.

Thanks!
